<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1320</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QLabel" name="src">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>100</y>
                        <width>631</width>
                        <height>451</height>
                    </rect>
                </property>
                <property name="frameShape">
                    <enum>QFrame::Box</enum>
                </property>
                <property name="text">
                    <string/>
                </property>
            </widget>
            <widget class="QLabel" name="dst">
                <property name="geometry">
                    <rect>
                        <x>670</x>
                        <y>100</y>
                        <width>631</width>
                        <height>451</height>
                    </rect>
                </property>
                <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                    </sizepolicy>
                </property>
                <property name="minimumSize">
                    <size>
                        <width>0</width>
                        <height>0</height>
                    </size>
                </property>
                <property name="frameShape">
                    <enum>QFrame::Box</enum>
                </property>
                <property name="text">
                    <string/>
                </property>
            </widget>
            <widget class="QWidget" name="verticalLayoutWidget">
                <property name="geometry">
                    <rect>
                        <x>100</x>
                        <y>10</y>
                        <width>1071</width>
                        <height>80</height>
                    </rect>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout"/>
            </widget>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>1320</width>
                    <height>22</height>
                </rect>
            </property>
            <widget class="QMenu" name="menuFile">
                <property name="title">
                    <string>File</string>
                </property>
                <addaction name="actionOpen"/>
            </widget>
            <widget class="QMenu" name="menuTransform">
                <property name="title">
                    <string>Transform</string>
                </property>
                <addaction name="actionRotate"/>
                <addaction name="actionTranslate"/>
                <addaction name="actionScale"/>
            </widget>
            <addaction name="menuFile"/>
            <addaction name="menuTransform"/>
        </widget>
        <action name="actionfgf">
            <property name="text">
                <string>fgf</string>
            </property>
        </action>
        <action name="actionopen_2">
            <property name="text">
                <string>open</string>
            </property>
        </action>
        <action name="actionOpen">
            <property name="text">
                <string>Open</string>
            </property>
        </action>
        <action name="actionRotate">
            <property name="text">
                <string>Rotate</string>
            </property>
        </action>
        <action name="actionTranslate">
            <property name="text">
                <string>Translate</string>
            </property>
        </action>
        <action name="actionScale">
            <property name="text">
                <string>Scale</string>
            </property>
        </action>
    </widget>
    <resources/>
    <connections/>
</ui>
